python/dasclient/dasclient.py

```diff
@@ -141,14 +139,19 @@ def _get(self, path, stream=False, **kwargs):
         self.logger.debug("Fail: " + response.text)
         raise DasClientException('Failed to call DAS web service.')
 
-        
-    def _post(self, path, payload):
+    
+    def _call(self, path, payload, method):
```

Comment:
**This is more of a suggestion for cases like this in the future**
This** might be a good candidate for enforcing keyword arguments and you can do by packing all positional arguments into a '*'. Like so:

```
    def _call(self, *, path=None, payload=None, method=None):
        ...
```

Obviously that requires updating the callers, but it looks like they're all within this module. Anyhow, the benefit will be as it gets used more and more, and if other arguments get added, it will reduce the risk of a subtle change causing mysterious errors from misaligned positionals.